services/viva-ms/lambdas/submitApplication.js

```diff
@@ -33,7 +32,15 @@ export async function main(event) {
 }
 
 async function sendApplicationsToViva(application) {
-  const { PK, answers, workflowId, pdf: pdfBinaryBuffer } = application;
+  const {
+    PK,
+    forms: {
+      [application.currentFormId]: { answers },
+    },
+    details: { workflowId },
+    pdf: pdfBinaryBuffer,
+  } = application;
+
```

Comment:
Maybe out of this scope. But why do we refer to cases here as applications? I get that viva use this naming convention. But why have we adpoted it here

Reply:
Då bör vi gå hela vägen och byta namn på lambdan. Så länge vi håller namn konsekvent. Case är i sig ett skitnamn då det är ett reserverat ord i javascript mfl.

Resonerar gärna vidare.

Reply:
Passar tillbaka och frågar om du testat o köra lambdan? Kanske mer relevant i detta läge... :)

Reply:
Tested and works fine (y)